web/controllers/game_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Yggdrasil.GameController do
+  use Yggdrasil.Web, :controller
+
+  alias Yggdrasil.Game
+
+  def index(conn, _params) do
+    games = Repo.all Game
```

Comment:
Yeah but the game engine will never need to worry about removing characters and such, that'll be a web thing. Also the web will need abilities to search but the game engine will most likely only perform id based lookups. Other administration things like deactivating old accounts and such, the game engine doesn't care about any of that. 

The issue with a character requiring updating multiple tables might be handled for us when Ecto hits 2.0, but either way I get that we might need to write a module to hide that stuff behind. I do question if everything the web API will need should be in `Yggdrasil`. So maybe I should ask do you see `Yggdrasil` encompassing everything?